Tools/_lust/replace_nt.py

```
@@ -7,17 +7,36 @@
+
+def replace_outside_placeholders(text: str) -> str:
+    parts = PLACEHOLDER_SPLIT.split(text)
+    for i, seg in enumerate(parts):
+        if seg.startswith('{') and seg.endswith('}'):
+            continue
+        for pattern, replacement in REPLACEMENTS.items():
+            seg = re.sub(pattern, replacement, seg, flags=re.IGNORECASE)
+        parts[i] = seg
+    return ''.join(parts)
+
 def process_ftl_files(locale_dir: Path):
```

Ruff (0.12.2)

Missing return type annotation for public function `process_ftl_files`

Add return type annotation: `None`

(ANN201)